modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/writer/interceptor/PageEntityExtensionWriterInterceptor.java

```diff
@@ -50,7 +50,10 @@ public class PageEntityExtensionWriterInterceptor implements WriterInterceptor {
 	public void aroundWriteTo(WriterInterceptorContext writerInterceptorContext)
 		throws IOException {
 
-		if (Page.class.isAssignableFrom(writerInterceptorContext.getType())) {
+		if (Page.class.isAssignableFrom(writerInterceptorContext.getType()) &&
```

Comment:
Can this break a preexisting use case? The condition is now more restrictive.

Reply:
Well, the condition I am adding is just to ensure that the next line could be executed and will not throw an NPE. I am not making the condition more restrictive, just avoiding NPE.

If something doesn't work now, it should be "exploding" actually 😅 
With this, will just continue with the execution